src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -235,9 +238,27 @@ def load_asfr(
 def load_sbr(
     key: str, location: str, years: Optional[Union[int, str, List[int]]] = None
 ) -> pd.DataFrame:
+    year_start, year_end = metadata.ARTIFACT_YEAR_START, metadata.ARTIFACT_YEAR_END
+
     sbr = load_standard_data(key, location)
+    lower_value = sbr.loc[(year_start, year_end, "lower_value"), "value"]
+    mean_value = sbr.loc[(year_start, year_end, "mean_value"), "value"]
+    upper_value = sbr.loc[(year_start, year_end, "upper_value"), "value"]
     sbr = sbr.reorder_levels(["parameter", "year_start", "year_end"]).loc["mean_value"]
-    return sbr
+
+    sbr_dist = sampling.get_truncnorm_from_quantiles(
+        mean=mean_value, lower=lower_value, upper=upper_value
+    )
+    sbr_draws = get_random_variable_draws(metadata.ARTIFACT_COLUMNS, key, sbr_dist)
+    sbr_draws = sbr_draws.values.flatten()
+    # Ensure shape is (1, NUM_DRAWS)
+    sbr_draws = sbr_draws.reshape(1, -1)
+
+    draw_columns = [f"draw_{i:d}" for i in range(data_values.NUM_DRAWS)]
+    sbr_draws_df = pd.DataFrame(sbr_draws, columns=draw_columns)
+    sbr_draws_df["year_start"] = year_start
+    sbr_draws_df["year_end"] = year_end
+    return sbr_draws_df.set_index(["year_start", "year_end"])
```

Comment:
Why is all this different?

Reply:
We now have variance instead of just the mean.